tests/acceptance/README.md

```diff
@@ -0,0 +1,52 @@
+# Acceptance Tests
+
+The following BATs tests can be used to test basic functionality of the GCP Secrets Engine.
+
+## Prerequisites
+
+* Clone this repository to your workstation
+* [Bats Core installed](https://bats-core.readthedocs.io/en/stable/installation.html#homebrew)
+* Docker
+* Vault CLI installed
+* GCP Project that has a service account w/ [required permissions](https://www.vaultproject.io/docs/secrets/gcp#required-permissions)
+
+### GCP Testing
+
+First, set the following env variables from your GCP project 
+
+* SERVICE_ACCOUNT_ID
+* PATH_TO_CREDS env variable pointing to service account credentials JSON file
+* GOOGLE_APPLICATION_CREDENTIALS
+* GOOGLE_PROJECT
+* GOOGLE_REGION
+
+Next, set the following environment variable to specify the version of Vault to test
+```bash
+$ export VAULT_IMAGE='hashicorp/vault:1.9.0-rc1'
+```
+
+Update the file `tests/acceptance/mybindings.hcl` with your GP project name for accurate
```

Comment:
```suggestion
Update the file `tests/acceptance/configs/mybindings.hcl` with your GP project name for accurate
```

Reply:
That's a great idea! Thanks for the tip, resolved in `a5b764e`